src/main/java/dev/jbang/cli/Init.java

```diff
@@ -31,15 +31,15 @@
 	static String[] knowneditors = { "code", "eclipse", "idea", "vi", "emacs", "netbeans" };
 
 	@CommandLine.Option(names = { "--template",
-			"-t" }, description = "Init script with a java class useful for scripting", defaultValue = "hello")
-	String initTemplate;
+			"-t" }, description = "Init script with a java class useful for scripting")
+	public String initTemplate;
 
 	@CommandLine.Option(names = {
 			"--force" }, description = "Force overwrite of existing files")
-	boolean force;
+	public boolean force;
```

Comment:
why do these need to be public now ?

Reply:
For testing. I'm testing in the configurations tests (which are not in the same package as the CLI commands) that they correctly pick up any configuration settings. Because they are in a different package I can only do that if the CLI commands have public fields.

Is there a issue with that? Having those fields public is not going to cause any problems AFAIK, right?
If yes, then we'd need to come up with a different way to run those tests.

Reply:
I don't see any test referring to them ?

Reply:
I would rather put the tests in the right package or have an accessor for the field if we absolutely need them.